setup.sh

```diff
@@ -266,9 +266,9 @@ echo -n "Building exonerate..."
 (
   if [ -e $SETUP_DIR/$CURR_TOOL.success ]; then
     echo -n " previously installed ..."
-  else [ $INST_METHOD -eq 2 ]; then
+  elif [ $INST_METHOD -eq 2 ]; then
 		echo " Skipping exonerate install ..."
```

Comment:
Hard tabs!!!, you should be using 2 space soft tabs in all file formats (not Makefile)

Reply:
sorry should have been looked more carefully

Reply:
removed hard tabs